rails-commenting.rb

```diff
@@ -53,15 +53,15 @@ def destroy
     if @post.destroy
       redirect_to blog_posts_path
     else
-      # 7)
+      # 7)if the destroy doesnt go through they will be redirected to he post they tried to delete.
       redirect_to blog_post_path(@post)
     end
   end
 
-  # 8)
+  # 8) this method can only be called  inside the class it is defined
   private
   def blog_post_params
-    # 9)
+    # 9)this makes it so the key pairs allowed will be the :title, :content
```

Comment:
Yes, when allowing content, the params.require makes it so you can only have .permit :title and :content